factory/config.go

```diff
@@ -51,6 +52,13 @@ type clusterConfig struct {
 	Arch             string
 }
 
+// GetConfigVarValue retrieves the string value of any variable from the tfvars file
+func GetConfigVarValue(g GinkgoTInterface, variable string) string {
```

Comment:
can we delete this since u are not using it? + i really dont see the need to attach ginkgo here if we can retrieve the values without this dependency as well.